code/src/main/java/com/onlinecontacttracing/servlets/get-data/PositiveUserInfoServlet.java

```diff
@@ -41,7 +41,11 @@ void useCredential(String userId, Credential credential, HttpServletResponse res
     contactInfo.join();
 
     // TODO Load PositiveUserContacts from objectify
-    // TODO call mergeContactListsFromPeopleAPI(calendarDataForPositiveUser.getContacts())
+    PositiveUserContacts p = new PositiveUserContacts(userId);
+    p.mergeContactListsFromCalendarAPI(calendarDataForPositiveUser.getContacts());
+    ofy().save().entity(p).now();
+
+    // TODO response.sendRedirect("/JSP/test.jsp?idToken=" + idTokenString);
```

Comment:
Should we implement this now?

Reply:
Or does this page not exist yet?

Reply:
merged with scopes branch so I implemented this